src/components/buttons/AddEmployeeButton/AddEmployeeButton.js

```diff
@@ -0,0 +1,23 @@
+import React from 'react';
+import './AddEmployeeButton.css';
+import { IoIosAddCircleOutline } from 'react-icons/io'
+
+const AddEmployeeButton = ({
+    text
+}) => {
+
+    function handleClickPopup() {
+
+    }
+
+    return (
+        <button className="addemployee-iconbtn" onClick={handleClickPopup}  >
```

Comment:
It is a small preference (and in no way necessary), but could our "own" class names be divided based on words? Such that `addemployee-iconbtn` becomes `add-employee-icon-btn`